[DOC-934] Clean up node agents (#30045)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-platform/administer-yugabyte-platform/uninstall-software.md

Reviewer: For systemd universes (system-level services), run the following commands:

type: docs
---

## Uninstall YugabyteDB Anywhere

If you used YBA Installer to install YugabyteDB Anywhere, you can use the `clean` command to uninstall the software. This removes the YugabyteDB Anywhere software, but keeps any data such as PostgreSQL or Prometheus information. Refer to [Clean](../../install-yugabyte-platform/install-software/installer/#clean-uninstall).

To completely eliminate all traces of YugabyteDB Anywhere and configuration, you should consider reinstalling the operating system image (or rolling back to a previous image, if available).

### Uninstall in Kubernetes environments

> 

Reviewer: We should probably add some guide about deleting any existing universes (if there are any) - Before uninstalling: If YugabyteDB Anywhere is managing universes, delete them as needed. Back up any configuration or data you want to keep. Uninstalling removes the YugabyteDB Anywhere application but does not automatically remove YugabyteDB clusters that were created through it. cc @nkhogen

You can uninstall YugabyteDB Anywhere in Kubernetes, as follows:

> 1. To remove YugabyteDB Anywhere, execute the following Helm command:

Reviewer: ```suggestion
1. To remove YugabyteDB Anywhere, run the following Helm command:
```

    ```sh
    helm uninstall yw-test -n yw-test
    ```

    `-n` option specifies the namespace scope for this request.

    You should see a message similar to the following, notifying you that the subject release has been removed:

    ```output

>     release "yw-test" uninstalled

Reviewer: ```suggestion
        release "<release-name>" uninstalled
```

    ```

> 2. Execute the following command to remove the `yw-test` namespace:

Reviewer: ```suggestion
1. Run the following command to remove the namespace:
```

    ```sh
    kubectl delete namespace yw-test
    ```

    You should see a message similar to the following:

    ```output
    namespace "yw-test" deleted
    ```

## Remove YugabyteDB components from nodes

> As described in [Eliminate an unresponsive node](../../manage-deployments/remove-nodes/), when a node enters an undesirable state, you can delete the node, with YugabyteDB Anywhere clearing up all the remaining artifacts except the `prometheus` and `yugabyte` user.

Reviewer: ```suggestion
As described in [Eliminate an unresponsive node](../../manage-deployments/remove-nodes/#eliminate-an-unresponsive-node), when a node enters an undesirable state, you can delete the node, with YugabyteDB Anywhere clearing up all the remaining artifacts except the `prometheus` and `yugabyte` user.
```

You can manually remove Yugabyte components from existing server images. Before attempting this, you have to determine whether or not YugabyteDB Anywhere is operational. If it is, you either need to delete the universe or delete the nodes from the universe.

### Delete on-premises database server nodes

You can remove YugabyteDB components and configuration from on-premises provider database server nodes as follows:

1. Log in to the server node as the `yugabyte` user.

1. Navigate to the `/home/yugabyte/bin` directory that contains a number of scripts including `yb-server-ctl.sh`. The arguments set in this script allow you to perform various functions on the YugabyteDB processes running on the node.

    If you cannot find the `bin` directory, it means YugabyteDB Anywhere already cleared it during a successful deletion of the universe.

> 1. For cron-based universes, run the following commands:

Reviewer: Stop YugabyteDB processes. For cron-based universes,  run the following from the home directory:
